### DISCUSSION TITLE
How do we use 'next export' with the new 'app' directory?

### DISCUSSION THREAD

**Discussion by PaulHaze** [vercel/next.js#42196] [Help] [answered]
So I am building a very simple landing page in Next 13 that won't really be using external API calls or anything like that so I would like to build a completely static final site using next build and next export. 

However I get an error which says 
```
No "exportPathMap" found in "/path/to/project/next.config.js". 
Generating map from "./pages"
```

As I am not using pages I am a bit confused how to move forward as the docs for exportPathMap are all for next 12 and assume you are using the 'pages' directory.

Does anyon have any advice on how to set this up using the 'app' directory?

**Comment by icyJoseph** [comment, marked as answer]
It is not yet supported, https://beta.nextjs.org/docs/app-directory-roadmap#planned-features:

> The app directory is in beta and many features are not yet supported. During the beta period, APIs could change based on community feedback. The following features are not yet supported:

And `next export` is listed as a non supported feature, **_yet_**.

**Reply by PaulHaze** [reply]
Got it, thanks. I'll stick with just using pages for now until things settle down a little with the '/app' directory....
